client/flutter/lib/pages/travel_advice.dart

```diff
@@ -56,55 +55,50 @@ class _TravelAdviceState extends State<TravelAdvice> {
     return SliverList(
         delegate: SliverChildListDelegate([
       Container(
-        color: Color(0xffD82037),
-        padding: EdgeInsets.symmetric(horizontal: 25, vertical: 10),
-        child: Html(
-          data: _adviceContent?.banner ?? "",
-          defaultTextStyle: TextStyle(
-            color: Colors.white,
-            // The Html widget isn't using the textScaleFactor so here's
-            // the next best place to incorporate it
-            fontSize: 18 * MediaQuery.textScaleFactorOf(context),
-            height: 1.2,
-          ),
-        ),
-      ),
-      Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 16.0, vertical: 10.0),
-        child: Semantics(
-          header: true,
-          child: Text(
-            S.of(context).travelAdvicePageListTitle,
-            style: TextStyle(
-                fontSize: 24,
-                fontWeight: FontWeight.w800,
-                color: Color(0xff050C1D)),
-          ),
-        ),
-      ),
+          padding: const EdgeInsets.all(26.0),
```

Comment:
Another super-nit: gutters/paddings should be in multiples of 4. Definitely not a blocker here, though

Reply:
Changed to 28.  However David's mock uses 26, 29, and more.